src/Features/LightLimitFix.cpp

```diff
@@ -1028,10 +1028,9 @@ void LightLimitFix::UpdateLights()
 	context->CSSetUnorderedAccessViews(0, 3, null_uavs, nullptr);
 }
 
-void LightLimitFix::Hooks::BSBatchRenderer_RenderPassImmediately::thunk(RE::BSRenderPass* Pass, uint32_t Technique, bool AlphaTest, uint32_t RenderFlags)
+bool LightLimitFix::BSBatchRenderer_RenderPassImmediately(RE::BSRenderPass* pass, uint32_t technique)
```

Comment:
Are you sure the original re was wrong here? Dropping args can result in weird behavior later if wrong.

Reply:
The hooks have been moved from LightLimitFix to Hooks.cpp so that ISS can use one of them. Since the compiler errors on unused parameters those have been removed.

Reply:
Compiler errors for unused parameters aren't a reason to change the RE. You can remove the parameter name and it won't complain.

If all this is doing is CheckParticleLights, why not just use that call where it's called later instead of keeping this weird butchered RE function name?

Reply:
Yeah easy enough I can directly call CheckParticleLights there. I left the name as is so it would still describe where it was hooking while reading through LightLimitFix but I can see the other side.